test/integration/gardenlet/controllerinstallation/controllerinstallation/controllerinstallation_suite_test.go

```diff
@@ -106,7 +105,10 @@ var _ = BeforeSuite(func() {
 	testClient, err = client.New(restConfig, client.Options{Scheme: kubernetes.GardenScheme})
 	Expect(err).NotTo(HaveOccurred())
 
-	testRunID = utils.ComputeSHA256Hex([]byte(uuid.NewUUID()))[:8]
+	// Prevent testRunID from being able to be interpreted as number, see https://github.com/gardener/gardener/issues/6786
```

Comment:
Is it possible to use 

```suggestion
	testRunID = testID + "-" utils.ComputeSHA256Hex([]byte(uuid.NewUUID()))[:8]
```

?

Some example from existing tests https://github.com/gardener/gardener/blob/9444417324f2843d3389a28a918239558bd69fb7/test/integration/controllermanager/seed/backupbucketscheck/backupbucketscheck_suite_test.go#L57 https://github.com/gardener/gardener/blob/9444417324f2843d3389a28a918239558bd69fb7/test/integration/controllermanager/seed/extensionscheck/extensionscheck_suite_test.go#L62

Reply:
Sure, should be possible